homeassistant/components/dwd_weather_warnings/config_flow.py

```diff
@@ -25,27 +29,72 @@ async def async_step_user(
         errors: dict = {}
 
         if user_input is not None:
-            region_identifier = user_input[CONF_REGION_IDENTIFIER]
+            valid_options = (CONF_REGION_IDENTIFIER, CONF_REGION_DEVICE_TRACKER)
```

Comment:
`exclusive_options` may be a better name.
Also, this should be moved to module level.

Reply:
Updated in commit b3c59088b6c776ec28905335f49a5c4ecdcde69b